UndertaleModLib/UndertaleChunks.cs

```diff
@@ -120,7 +117,7 @@ public class UndertaleChunkEXTN : UndertaleListChunk<UndertaleExtension>
 
         internal override void UnserializeChunk(UndertaleReader reader)
         {
-            if (reader.undertaleData.GMS2_3)
+            if (reader.undertaleData.IsVersionAtLeast(2, 3) && !reader.undertaleData.IsVersionAtLeast(2022, 6))
```

Comment:
Is a 2022.6 check necessary?

Reply:
Yes. That code block is to set the version to 2022.6, so if the versions n has already been set higher it would be redundant.